tools/licenses/lib/filesystem.dart

```diff
@@ -177,6 +177,8 @@ FileType identifyFile(String name, Reader reader) {
     // Machine code
     case '.so': return FileType.binary; // ELF shared object
     case '.xpt': return FileType.binary; // XPCOM Type Library
+    // Graphics code
+    case '.glsl': return FileType.text;
```

Comment:
Should the `.spvasm` files be treated the same?

Reply:
Yes, I will need to update the license golden next